win32_build.sh

```diff
@@ -185,12 +185,17 @@ if [[ -n $BUILD_ZIP ]]; then
     # Use a temp directory, in order to create a clean zip file
     ZIP_TMPDIR=$(mktemp -d win_binaries.XXXXX)
     if [[ -n $STRIP_ZIPPED ]]; then
-        rm -rf $strippedBinDir
-        echo "Copying binaries to $strippedBinDir."
-        cp -r $binDir $strippedBinDir
         echo "Stripping debug symbols from binaries."
-        $MINGW_STRIP $strippedBinDir/*.exe \
-                     $strippedBinDir/*.dll
+        rm -rf $strippedBinDir; mkdir $strippedBinDir
```

Comment:
this file is not a CMake script. please update the title of the commit message accordingly.

Reply:
> please s/build/cmake/ in the title. we don't have subdirectory of a component named "build".

which should be the component? can I use "build:" or should I skip the component?

Reply:
> can I use "build:" 

i don't think so.`win32*.sh` would suffice. 

> or should I skip the component?

please don't. the title of a commit helps the maintainer to pin/search commit quickly. i think we've been discussing on this topic more than one time in the past.